pkg/volume/cinder/attacher_test.go

```diff
@@ -700,7 +700,7 @@ func (testcase *testcase) DisksAreAttachedByName(nodeName types.NodeName, volume
 		return areAttached, errors.New("unexpected DisksAreAttachedByName call: wrong instanceID")
 	}
 
-	klog.V(4).Infof("DisksAreAttachedByName call: %v, %s, returning %v, %v", volumeIDs, nodeName, expected.areAttached, expected.ret)
+	klog.V(4).InfoS("DisksAreAttachedByName call volumeIDs and nodeName, returns areAttached and ret", "volumeIDs", volumeIDs, "nodeName", nodeName, "areAttached", expected.areAttached, "ret", expected.ret)
```

Comment:
What is `ret` here?
```suggestion
	klog.V(4).InfoS("DisksAreAttachedByName call volumeIDs and nodeName, returns areAttached and ret", "volumeIDs", volumeIDs, "node", klog.KRef("", nodeName), "areAttached", expected.areAttached, "ret", expected.ret)
```

Reply:
Hi, @serathius. Looking up for variable `expected`, it is defined as `type disksAreAttachedCall struct`. In this struct, `ret` is defined as `error`. Can we modify `ret` to `err`?